oak-search/src/main/java/org/apache/jackrabbit/oak/plugins/index/search/spi/editor/FulltextDocumentMaker.java

```diff
@@ -336,6 +338,32 @@ private boolean indexProperty(String path,
         return dirty;
     }
 
+    private String getValueAsTruncatedString(Object val) {
+        String value = val.toString();
+        return value.length() <= 128 ? value
+            : value.substring(0, 128) + " ... [" + value.hashCode() + "]";
```

Comment:
What is the use case of printing the hascCode? In what situations will it be useful? I can't think of any right now.

Reply:
Just in case it would be useful? Put it mostly there to have a sense if anyone might think it would be useful. If not, can just remove it.

Reply:
👍

Reply:
Okay done!